Source/Lib/Codec/EbModeDecisionConfigurationProcess.c

```diff
@@ -2100,6 +2097,9 @@ void* ModeDecisionConfigurationKernel(void *inputPtr)
                 EbPostFullObject(encDecTasksWrapperPtr);
             }
         }
+#if DEADLOCK_DEBUG
+        SVT_LOG("POC %lu MDC OUT \n", pictureControlSetPtr->pictureNumber);
```

Comment:
move to right after EbPostFullObject?

Reply:
ModeDecisionConfiguration posts tile group(s) of a picture to EncDec. But DEADLOCK_DEBUG is used to track picture level, not tile, segment or LCU level.

Reply:
I see. thank you.